src/test/java/com/ithaca/user/UserTest.java

```diff
@@ -0,0 +1,30 @@
+package com.ithaca.user;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by David on 3/24/16.
+ */
+public class UserTest {
+
+    @Test
+    public void testUser() {
+        // Make a user with no construction values
+        User userEmpty = new User();
+        Assert.assertNotNull(userEmpty);
```

Comment:
So you never actually have to declare a test class. You simply run them. It can be done command line, but Idea makes it easy by just clicking the little thing next to the @Test notation called run. It then goes through and runs all of the tests.